next/modules/common/Button.tsx

```diff
@@ -82,7 +82,7 @@ const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, PolymorphicProp
                 'w-full lg:w-auto': mobileFullWidth,
 
                 // text colors
-                'text-white': variant === 'primary' || variant === 'plain-white',
```

Comment:
`plain-white` is this style: https://www.figma.com/file/CY6Mh2f0SXJhBMY74HdS03/MKB?node-id=79%3A657&t=PRamF6E6WVSaLsAA-0

Please create new variant and style it as needed, eg. `variant="carousel"`